Yoast/javascript
# Add full text tests for Catalan

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Catalan papers and specs to check the assessments functionality for Catalan.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and make sure all tests pass

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-638
